src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -69,21 +69,25 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE_SUMMARY = "Produces a file containing summary alignment metrics from a SAM or BAM.";
-    static final String USAGE_DETAILS = "<br />" +
+    static final String USAGE_SUMMARY = "Produces a file containing summary alignment metrics from a SAM or BAM file";
+    static final String USAGE_DETAILS = "Using read outputs from HTS technologies, this tool provides detailed metrics " +
+            "regarding the quality of the " +
+            "reads that passed machine signal-to-noise threshold quality filters (Illumina) as well as values representing the " +
+            "degree of alignment to a reference sequence.  " +
+            "Please see: <br />http://broadinstitute.github.io/picard/picard-metric-definitions.html#AlignmentSummaryMetrics" +
```

Comment:
Now reads, "as well as values that represent the quality of read alignments to a reference sequence.  " +